src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -2450,6 +2450,107 @@ TopoShape& TopoShape::makeElementOrderedWires(const std::vector<TopoShape>& shap
     return makeElementCompound(wires, nullptr, SingleShapeCompoundCreationPolicy::returnShape);
 }
 
+bool TopoShape::_makeElementTransform(const TopoShape& shape,
+                                      const Base::Matrix4D& rclTrf,
```

Comment:
What is "rclTrf" short for? And it doesn't seem to match the .h file, though the linter is not complaining.

Reply:
red colored liquid Transformation?  Yeah, that's weird.